lib/util/trip/trip_overview.dart

```diff
@@ -15,28 +15,32 @@ class TripOverview extends StatelessWidget {
       mainAxisAlignment: MainAxisAlignment.spaceBetween,
       children: [
         Expanded(
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Text(trip.start),
-              Text(
-                localeManager.formatTime(trip.startTime),
-                style: DefaultTextStyle.of(context).style.copyWith(fontWeight: FontWeight.w700),
-              )
-            ],
+          child: MergeSemantics(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(trip.start),
+                Text(
+                  localeManager.formatTime(trip.startTime),
+                  style: DefaultTextStyle.of(context).style.copyWith(fontWeight: FontWeight.w700),
+                )
+              ],
+            ),
           ),
         ),
         const Icon(Icons.arrow_forward_rounded),
         Expanded(
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.end,
-            children: [
-              Text(trip.end),
-              Text(
-                localeManager.formatTime(trip.startTime),
-                style: DefaultTextStyle.of(context).style.copyWith(fontWeight: FontWeight.w700),
-              )
-            ],
+          child: MergeSemantics(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.end,
+              children: [
+                Text(trip.end),
+                Text(
+                  localeManager.formatTime(trip.startTime),
+                  style: DefaultTextStyle.of(context).style.copyWith(fontWeight: FontWeight.w700),
+                )
+              ],
+            ),
           ),
         ),
       ],
```

Comment:
the seatindicator should probably be changed to have a label "10 of 6 seats", currently it says "10 6 seats" and it's not easy to understand

Reply:
Yeah right that is a bit weird, I'll change that